pageserver/src/pgdatadir_mapping.rs

```diff
@@ -2667,7 +2700,7 @@ pub struct DatadirModificationStats {
 /// timeline to not miss the latest updates.
 #[derive(Clone, Copy)]
 pub enum Version<'a> {
-    Lsn(Lsn),
+    Lsns(LsnRange),
```

Comment:
nit: `Lsns` is a bit vague. I'd go for `LsnRange` or `Specific`.

Reply:
Renamed